MSGraphSDKNXOAuth2/Auth/NXOAuth2AuthenticationProvider.m

```diff
@@ -188,9 +188,17 @@ - (void)loginWithViewController:(UIViewController*)viewController
     }];
 }
 
-//- (BOOL)loginSilent {
-//    // Placeholder: Load an existing account from the account store
-//}
+- (BOOL)loginSilent {
+    // Placeholder: Load an existing account from the account store
+    int accountCount = [[NXOAuth2AccountStore sharedStore] accounts].count;
+    
+    if (accountCount == 0) {
+        return NO;
+    } else {
+        self.userAccount = [[NXOAuth2AccountStore sharedStore] accounts][accountCount - 1];
```

Comment:
Nope it does not. 
In fact, I believe it is using a dictionary with account identifier. We can be more "accurate" by storing "last logged in" identifier and using that value to really fetch the last successful login. 

I just thought as a sample, with an assumption of a single user account login, this would suffice. 

Let me know.